assets/eip-7092/ERC7092.sol

```diff
@@ -155,15 +137,11 @@ contract ERC7092 is IERC7092, BondStorage {
         return true;
     }
 
-    function approveAll(address _spender) external external returns(bool) {
```

Comment:
@Pandapip1 ,

All functions that allow to manage all tokens of an account have been removed, since they could be a severe security issue. That why functions like `approveAll`, `transferAll`, etc. have been removed.